extensions/amp-instagram/1.0/storybook/Basic.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as Preact from '../../../../src/preact';
+import {Instagram} from '../instagram';
+import {number, text, withKnobs} from '@storybook/addon-knobs';
+import {withA11y} from '@storybook/addon-a11y';
+
+export default {
+  title: 'Instagram',
+  component: Instagram,
+  decorators: [withA11y, withKnobs],
+};
+
+export const _default = () => {
+  const width = number('width', 400);
+  const height = number('height', 600);
+  return (
+    <Instagram
+      shortcode="B8QaZW4AQY_"
+      width={width}
+      height={height}
+    ></Instagram>
+  );
+};
+
+export const withCaption = () => {
+  const width = number('width', 400);
+  const height = number('height', 600);
+  return (
+    <Instagram shortcode="B8QaZW4AQY_" captioned width={width}></Instagram>
+  );
+};
+
+export const responsive = () => {
+  const width = number('width', 400);
+  const height = number('height', 600);
+  return (
+    <Instagram
+      shortcode="B8QaZW4AQY_"
+      captioned
+      width={width}
+      layout="responsive"
+    ></Instagram>
+  );
```

Comment:
What do you think about turning `withCaption` (`captioned`) and `responsive` to `bool()` knobs as opposed to separate stories?